### ISSUE TITLE
Devices & Presets modal should disappear on selection

### ISSUE THREAD

**Issue by wesort** [emmetio/re-view#17]
Would be nice if the modal would disappear when a selection has been made from the list as it covers the viewport previews. 

Workaround:
- click on the space around the preview viewport
##

**Comment by sergeche** [comment]
This is intentional behaviour so users can play around with different presets and devices.

**Comment by wesort** [comment]
Fair enough for the Devices list. But I find it frustrating though when I've chosen a "Device Preset" and then need to click off. The whole point of the preset is a quick route to a few devices. 

Mapping the `esc` key to close the modal would be nice too.
